Makefile.in

```diff
@@ -14,7 +14,9 @@ CPU = @target_cpu@
 TARGET = @target_alias@
 SYMBOL_PREFIX = @SYMBOL_PREFIX@
 CFLAGS = @CFLAGS@
-INCLUDES = -I$(srcdir) @bfd_include_dir@ @binutils_include_dir@ @zlib_include_dir@
+INCLUDES = -I$(srcdir) -Ibfd-headers @zlib_include_dir@
+BFD_INCLUDE_DIR = $(shell cd "@bfd_include_dir@" && pwd)
```

Comment:
i was talking about the shell code in the `bfd-headers/.stamp` target, not global make variables.  that's why i wrote a shell snippet and didn't write make vars.